### PR TITLE
Fix breaking changes from Kord Kotlin/JS support

### PR DESCRIPTION
**Pull request by lukellmann**
see https://github.com/kordlib/kord/pull/775

### CODE REVIEW

File: kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Permissions.kt [hunk 1]
```diff
@@ -57,6 +57,8 @@ public fun Permission.toTranslationKey(): String? = when (this) {
 
     Permission.UseExternalStickers -> "permission.useExternalStickers"
     Permission.UseEmbeddedActivities -> "permission.useEmbeddedActivities"
+    Permission.ViewCreatorMonetizationAnalytics -> TODO()
+    Permission.UseSoundboard -> TODO()
```
Comment (lukellmann) [inline kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Permissions.kt]: it's a draft pr because i don't know what belongs here

Comment (NoComment1105) [reply kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Permissions.kt]: Looks to just be the translation key, i would assume just
```suggestion
    Permission.ViewCreatorMonetizationAnalytics -> "permission.viewCreatorMonetizationAnalytics"
    Permission.UseSoundboard -> permission.useSoundboard
```
then add to the translation files those lines. Just add it to the root strings.properties i believe

Comment (gdude2002) [reply kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Permissions.kt]: That one and the UK one yeah

Comment (lukellmann) [reply kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Permissions.kt]: done

**Review by gdude2002 — APPROVED** [review]

**Comment by gdude2002** [comment]
Thanks for that!

**Pull request merged**